lib/runner/cli/clirunner.js

```diff
@@ -159,7 +163,7 @@ CliRunner.prototype = {
       this.test_settings.globals = globals;
 
     } catch (err) {
-      throw new Error('External global file could not be located - using '+ this.settings.globals_path + '.');
+      throw new Error('Error reading external global file: ' + err);
```

Comment:
I would prefer just rethrowing `err` instead, like so: `throw err;` — because then the stack trace will be to the problematic line in your own `globals.js` file, so you'll see exactly where in your file the bug is.

By wrapping `err` in a text message and throwing a new error, the stack trace will be to `clirunner.js` instead which is not so very informative.